uvicorn/config.py

```diff
@@ -72,6 +84,12 @@ def __init__(
         timeout_notify=30,
         callback_notify=None,
         install_signal_handlers=True,
+        keyfile=None,
+        certfile=None,
+        ssl_version=ssl.PROTOCOL_TLS,
+        cert_reqs=ssl.CERT_NONE,
+        ca_certs=None,
+        ciphers="TLSv1",
```

Comment:
Again, I think it might be a good idea for us to prefix all SSL options with "ssl_"